src/app/view/perspectives/smartdoc/text/attribute.blot.ts

```diff
@@ -17,21 +17,22 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-declare const QuillEditor: any;
-const Parchment = QuillEditor.import('parchment');
-
-let Embed = QuillEditor.import('blots/embed');
+import EmbedType from 'quill/blots/embed';
+import * as quillEd from 'quill';
 
+const Parchment = quillEd['import']('parchment');
+let Embed: EmbedType = quillEd['import']('blots/embed');
 let Color = new Parchment.Attributor.Style('color', 'color', {
   whitelist: ['#18bc9c']
 });
 Parchment.register(Color);
 
 export class AttributeBlot extends Embed {
 
-  public static blotName = 'attribute';
-  public static className = 'attribute';
-  public static tagName = 'span';
+  public blotName = 'attribute';
+  public className = 'attribute';
+  public tagName = 'span';
+  public domNode;
```

Comment:
It seems that these changes prevents this custom blot from being used in Quill editor and cause an exception at runtime. Quill is expecting these fields to be static. I have managed to fix it by referring to them using `super.field` but it is not the best solution.

Do you have any idea how we can bring typed Quill classes here so our compiler won't complain? Here is the class hierarchy of the embed blot:

- [EmbedBlot](https://github.com/quilljs/parchment/blob/master/src/blot/embed.ts)
- [LeafBlot](https://github.com/quilljs/parchment/blob/master/src/blot/abstract/leaf.ts)
- [ShadowBlot](https://github.com/quilljs/parchment/blob/master/src/blot/abstract/shadow.ts)